graphql_ws/tornado.py

```diff
@@ -0,0 +1,114 @@
+from inspect import isawaitable
+
+from asyncio import ensure_future, wait, shield
```

Comment:
And all this...

Reply:
I think Tornado also offers something similar to `ensure_future` (not sure about `wait` and `shield` tho)